assets/javascripts/discourse/templates/components/chat-selection-manager.hbs

```diff
@@ -0,0 +1,25 @@
+<div class="chat-selection-management">
```

Comment:
is there any css associated with it?

Reply:
Yes there is mainly around the buttons and the success message shown on copy quote to clipboard

Reply:
So maybe we create an associated css files and move stuff related to this inside please